venia-integration-tests/src/tests/wishList/e2eTests/singleWishlistAddRemoveProduct.spec.js

```diff
@@ -0,0 +1,88 @@
+import {
+    accountAccess as accountAccessFixtures,
+    myAccountMenu as myAccountMenuFixtures,
+    categoryPage as categoryPageFixtures,
+    homePage as homePageFixtures,
+    wishlist as wishlistFixtures,
+    productPage as productPageFixtures
+} from '../../../fixtures';
+import {
+    categoryPage as categoryPageActions,
+    myAccountMenu as myAccountMenuActions,
+    productPage as productPageActions,
+    wishlist as wishlistActions
+} from '../../../actions';
+import {
+    myAccountMenu as myAccountMenuAssertions,
+    wishlist as wishlistAssertions
+} from '../../../assertions';
+
+const {
+    firstName,
+    lastName,
+    accountEmail,
+    accountPassword
+} = accountAccessFixtures;
+const { wishlistPage } = myAccountMenuFixtures;
+const { categorySweaters, productCarinaCardigan } = categoryPageFixtures;
+const { homePage } = homePageFixtures;
+const { wishistRoute } = wishlistFixtures;
+const { productValeriaTwoLayeredTankUrl } = productPageFixtures;
+
+const { goToMyAccount } = myAccountMenuActions;
+const { removeProductFromWishlist } = wishlistActions;
+const { addProductToWishlistFromCategoryPage } = categoryPageActions;
+const { addProductToWishlistFromProductPage } = productPageActions;
+
+const { assertCreateAccount } = myAccountMenuAssertions;
+const {
+    assertWishlistHeading,
+    assertEmptyWishlist,
+    assertProductInWishlist
+} = wishlistAssertions;
+
+// TODO add tags CE, EE to test to filter and run tests as needed
```

Comment:
Is this something that should get done before we merge this? If not, can you open a ticket and add to the comment?